sentinel-overlay/datacollection-config.xml

```diff
@@ -0,0 +1,12 @@
+<datacollection-config xmlns="http://xmlns.opennms.org/xsd/config/datacollection" rrdRepository="/Users/jesse/git/opennms/target/opennms-31.0.3-SNAPSHOT/share/rrd/snmp/">
```

Comment:
@j-white Just seeing this here: opennms-31.0.3-SNAPSHOT. Is this a problem with 32.0.1?